src/js/views/patients/patient/sidebar/sidebar_views.js

```diff
@@ -118,9 +129,10 @@ const sidebarWidgets = {
       const displayOptions = this.getOption('display_options');
 
       const value = key ? propertyOf(fieldValue)(key.split('.')) : fieldValue;
+      const opts = key ? propertyOf(displayOptions)(key.split('.')) : null;
```

Comment:
I think this widget is:
```js
templateContext() {
  const defaultHtml = this.getOption('default_html');
  const value = getFieldValue(this.model.getFields(), this.getOption('field_name'), this.getOption('key'));

  if (!value) return { defaultHtml };
  const displayOptions = this.getOption('display_options');

  return {
    displayValue: displayOptions[value] || value,
    defaultHtml,
  };
}
```

Reply:
yeah I tried that and it didn't work with nested `displayOptions` keys IIRC. I'll give it another shot though

Reply:
Oh display options themselves shouldn't be nested. only the patient field values will be nested

Reply:
The problem with nesting and the optionsWidget was this: `const value = key ? propertyOf(fieldValue)(key) : fieldValue;`. where `key` could not be `deeply.nested.key` for the `fieldValue` because it was missing the `.split`